libs/portal-integration-angular/assets/i18n/de.json

```diff
@@ -15,6 +15,17 @@
       "NO": "Nein",
       "LEFT": "Links",
       "RIGHT": "Rechts"
+    },
+    "ARIA_LABELS": {
+      "UP": "Spalte nach oben verschieben",
+      "TOP": "Spalte an den Anfang verschieben",
+      "DOWN": "Spalte nach unten verschieben",
+      "BOTTOM": "Spalte ans Ende verschieben",
+      "RIGHT": "Spalte als aktiv setzen",
```

Comment:
Does this naming makes sense?
I would expect:
"RIGHT": "Spalte nach rechts setzen",

OR

"ACTIVE": "Spalte als aktiv setzen"

Reply:
So this is specific to OCX_CUSTOM_GROUP_COLUMN_SELECTOR where you select active an inactive columns by moving them from left (inactive columns) to right (active columns)

Reply:
Ahh okay. Then this is fine

Reply:
Actually german translations have the wrong naming. Right and left should be swapped.